khal/icalendar.py

```diff
@@ -46,7 +47,19 @@ def split_ics(ics, random_uid=False, default_timezone=None):
     :rtype list:
     """
     cal = cal_from_ics(ics)
-    tzs = {item['TZID']: item for item in cal.walk() if item.name == 'VTIMEZONE'}
+    tzs = {}
+
+    # Since some event could have a Windows format EG : 'New Zealand Standard Time'
+    # for 'Pacific/Auckland' in Olson format, we should get the last format and put
+    # it in tzs key to avoid warning in ics_from_list (issue #876)
```

Comment:
Alternative wording?
```suggestion
    # Since some events could have a Windows format timezone (e.g. 'New Zealand
    # Standard Time' for 'Pacific/Auckland' in Olson format), we convert any
    # Windows format timezones to Olson.
```